components/rating.css

```diff
@@ -0,0 +1,14 @@
+@import url(/stylesheets/main.css);
+
+.ratings-container{
+    width: 30rem;
+    background-color: hsl(var(--primary-lighter));
+    display: flex;
+    justify-content: center;
+    color: rgb(255,164,29);
+}
+
+.numbered-rating{
+    background-color: hsl(var(--green-color));
```

Comment:
Same as above

Reply:
addressed your concerns in the above comments.